src/main/java/de/leonhard/storage/util/FastStringWriter.java

```diff
@@ -1,21 +1,24 @@
 package de.leonhard.storage.util;
 
-import java.io.Writer;
 import lombok.Getter;
 import lombok.NonNull;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.Writer;
 
 /**
  * A Writer writing in a StringBuilder. This is NOT Thread safe.
  *
  * @author TheElectronWill Edited by JavaFactoryDev.
  */
 @Getter
+@SuppressWarnings("unused")
```

Comment:
Why is this unused?

Reply:
There are some "unused" method warnings.